arboretum/ibm_cloud/README.md

```diff
@@ -62,6 +62,49 @@ to include the fetchers and checks from this library in your downstream project.
    from arboretum.ibm_cloud.fetchers.fetch_cluster_list import ClusterListFetcher
    ```
 
+### Cluster Resource
+
+* Class: [ClusterResourceFetcher][fetch-ibm-cloud-cluster-resource]
+* Purpose: Write the resources of IBM Cloud Kubernetes clusters to the evidence locker.
+* Behavior: Retrieve the resources of IBM Cloud Kubernetes clusters listed by [cluster list fetcher][fetch-cluster-list]. For IBM Cloud Kubernetes Service (IKS) clusters, the fetcher [downloads cluster config via IBM Cloud API][ibm-cloud-download-config] and extract the access token from the config. For IBM Cloud Red Hat Kubernetes Service (ROKS) clusters, the fetcher retrieves the access token using [the OAUTH server][ibm-cloud-roks-oauth] of the cluster. TTL is set to 1 day.
+
+* Configuration elements:
+  * `org.ibm_cloud.accounts`
+    * Required
+    * List of accounts (string)
+    * Each account is an arbitrary name describing the IBM Cloud account. It is used to match to the token provided in the
+      credentials file in order for the fetcher to retrieve content from IBM Cloud for that account.
+  * `org.ibm_cloud.cluster_resources.target_resource_types`
+    * Optional
+    * List of resource types as strings
+    * See the document of the [Kubernetes resource fetcher][fetch-kube-cluster-resource] for details.
```

Comment:
?

This link is sending you to the code for the Kubernetes cluster resource fetcher.  It's not clear why.

Reply:
I changed the link target to the description in the README file of the Kubernetes category.  Does this make sense for you?

Reply:
The config of the kubernetes fetcher isn't even the same as this one.  This is super confusing.

If target_resource_types are optional then the reader needs to know what the default is if the config is not provided.